source/ConditionsStore.h

```diff
@@ -19,13 +19,13 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include <functional>
 #include <initializer_list>
 #include <map>
-#include <string>
+
+#include "ConditionEntry.h"
```

Comment:
This should go above the std library includes.

Reply:
Yes, fixed in last commit.